.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,41 @@
+# Dependabot files PRs by pushing branches named
+# "dependabot/{ecosystem}/{depname}".  What it pushes to those
+# branches is pretty naive; it doesn't support us telling it that it
+# needs to run "make generate" after making an edit.
+#
+# But we can work around that by creating a workflow run "make
+# generate" on those branches!
+name: dependabot
+"on":
+  push:
+    branches:
+      - 'dependabot/**'
```

Comment:
Should we limit this to only be on created so that it doesn't run if add additional commits on the PR?

Reply:
> but dependabot updates its PRs I think, if a new version comes out before the original PR is merged

that is correct